<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenshotDialog</class>
 <widget class="QDialog" name="ScreenshotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>378</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>339</width>
    <height>339</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>339</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Screenshot</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/img/128/pixmate.png</normaloff>:/img/128/pixmate.png</iconset>
  </property>
  <widget class="QLabel" name="screenshotLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>341</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="shareButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Share</string>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/ic_cloud_upload_black_24dp.ico</normaloff>:/img/ic_cloud_upload_black_24dp.ico</iconset>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/ic_desktop_windows_black_24dp.ico</normaloff>:/img/ic_desktop_windows_black_24dp.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/ic_close_black_24dp.ico</normaloff>:/img/ic_close_black_24dp.ico</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="shareGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string> Share settings</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QSpinBox" name="ttlTimeSpinBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.SF NS Text</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Time in minutes</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>32500</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="ttlTimeCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Limit time to (minutes)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ttlViewsCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Limit to (views)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ttlViewsSpinBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.SF NS Text</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Time in minutes</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>32500</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="customTitleLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Custom title (optional)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
